src/work.py

```diff
@@ -0,0 +1,146 @@
+""" Create simulator from real life. This can be booking room in hotel,
+visit to casino, visit to bar. Create 3-4 objects, that can describe situation.
+Objects should contain attributes and methods to simulate some use cases.
+Completed program should print object states, it actions (methods) and
+objects interaction.
+"""
+
+from random import randint
+
+
+class CreateHuman(object):
+    def create_human(self, name, surname):
+        self.name = name
+        self.surname = surname
+        self.salary = randint(1, 15)
+        self.money = 0
+        self.company = False
+        self.company_name = False
```

Comment:
why do we need this name if it would be in company instance?

Reply:
We don't need it.